lib/cli/ui/widgets/status.rb

```diff
@@ -0,0 +1,55 @@
+# frozen-string-literal: true
+require('cli/ui')
+
+module CLI
+  module UI
+    module Widgets
+      class Status < Widgets::Base
```

Comment:
This whole class feels like a funny fit in cli-ui.  It's pretty specific to our use-case.  I wonder if it would be better to have a registry of widgets, much like cli-kit does for commands.  As is, adding a widget means a PR here, or else a monkey-patch, which is far from ideal.